docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,94 @@
+7. (This is optional) Set an expiration date for the payment method. This is especially useful when a budget expires, e.g. at the end of the accounting year.
+8. Additionally, you can enter custom settings to inject metadata into the payment method. This is useful when you want to enhance the payment method with organizational details like the cost center number or the business unit.
+9. Click 'Save' and your new payment method will be available to the meshCustomer it was created in!
+
+### Creating a payment method via the meshstack API
+
+As automation matters, there is also the possibility to create payment methods via the meshstack API. To find the API documentation, open your meshstack and click on 'API' at the bottom of the meshPanel. A new page will open and on the left-hand side, you should see `meshPaymentMethod` listed under `meshObject Import`. Click on it and read the documentation to understand how you can create a payment method via the API.
+
+## The Payment Method Lifecycle: Assigning to Projects
+
+Now that the Partner Admin has created one or more payment methods for a meshCustomer, we are ready to link a payment method to one or more projects.
+
+First, check if the payment method is correctly created in the meshCustomer. You can do so by navigating to the meshCustomer Account area. In order to do that you need to have Customer Admin rights within the respective customer. On the left-hand side, there should be an item called 'Payment Methods' which you can navigate to. All payment methods that are created and assigned to your current meshCustomer are shown here.
+
+The payment methods that are assigned can be used for both existing projects and new projects.
+
+### Applying a payment method to an existing project
+
+In the Customer Account area, open 'Projects' on the left. Then click on the 'Edit project' button on the right for the project you would like to add a payment method. In the 'Edit Project' screen, you'll see the selected payment methods on the right. As explained at the top of the page, you also have the ability to specify a Substitute Payment Method. A Substitute Payment Method is useful when working with expiring cost centers or budgets. This makes sure that when the active payment method expires, it will activate the substitute payment method.
+
+![Set Payment Method in Project Edit screen](assets/payment_methods/payment_method_selection_project_edit.png)
+
+Select the Active Payment Method of choice, and optionally a Substitute Payment Method if that is desired. Save the project and the payment method(s) will be assigned to the project.
+
+### Applying a payment method to a new project
+
+Create a new project at the top of the meshPanel. Follow the project creation flow until you enter the 'Billing Information' page.
+
+![Set Payment Method in Project Create screen](assets/payment_methods/payment_method_selection_project_create.png)
```

Comment:
The big headlines within the images are bigger than those of the actual documentation, this somehow prevents me from parsing the structure of the docs correctly.

Reply:
Added a new screenshot and reduced the heading size.